### ISSUE TITLE
Consider moving Miscellaneous ops to CHLO

### ISSUE THREAD

**Issue by burmako** [openxla/stablehlo#1570]
More specifically, I mean the following ops: `batch_norm_grad`, `batch_norm_inference`, `batch_norm_training`, `cholesky`, `fft`, `rng`, `rng_bit_generator`, `triangular_solve` - all of which feel a bit out of place in the StableHLO opset (they all feel like composite ops which is not something that StableHLO encourages).

Furthermore, all of these ops except one have expander passes implemented in the XLA compiler, which further suggests that they would be a better fit for CHLO rather than StableHLO:
  * `batch_norm_grad`, `batch_norm_inference` and `batch_norm_training`: [batchnorm_expander.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/batchnorm_expander.h).
  * `cholesky`: [cholesky_expander.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/cholesky_expander.h).
  * `rng`: [rng_expander.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/rng_expander.h).
  * `rng_bit_generator`: [rng_bit_generator_expander.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/rng_bit_generator_expander.h).
  * `triangular_solve`: [triangular_solve_expander.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/triangular_solve_expander.h).

FftOp is a bit of a mystery to me. It doesn't have an expander, but it also feels out of place. Maybe I'm just wrong, and it should join SortOp in the Data Movement category.
